tests/test_flask_tracing.py

```diff
@@ -111,7 +111,6 @@ def test_span_tags(self):
             tags.HTTP_METHOD: 'GET',
             tags.SPAN_KIND: tags.SPAN_KIND_RPC_SERVER,
             tags.HTTP_URL: 'http://localhost/another_test_simple',
-            'is_xhr': 'False',
```

Comment:
Not sure why but this tag isn't present in the test span anymore. Probably latest version of flask doesn't add it.

Reply:
Looks like they obsoleted it in werkzeug, which was even a surprise to the flask creator.  Mind updating the tags setting for the `/another_test_simple` handler so we don't lose coverage?  https://github.com/signalfx/python-flask/pull/2/files#diff-3360ab3a009cff0d6f43ecd38cfd2781R39

Reply:
Updated.